client/blocks/sharing-preview-pane/index.jsx

```diff
@@ -132,12 +135,14 @@ const mapStateToProps = ( state, ownProps ) => {
 	const seoTitle = getSeoTitle( state, 'posts', { site, post } );
 	const currentUserId = getCurrentUserId( state );
 	const connections = getSiteUserConnections( state, siteId, currentUserId );
+	const connectedServices = map( connections, 'service' );
```

Comment:
Using lodash is totally fine here, but `Array.prototype.map` is available in IE9+ and is already used in this file so we could use that instead.